index.html

```diff
@@ -430,22 +434,33 @@ <h2 class="title is-2">Contact us</h2>
             <div class="field">
               <div class="control">
                 <label class="checkbox">
-                  <input type="checkbox">
-                  I agree to the <a class="best-company-link" href="terms-conditions" target="_blank" onclick="return false;">terms and conditions
+                  <input type="checkbox" aria-placeholder="Confirm agreement" aria-describedby="linkHelp">
+                    I agree to the 
+                    <a class="best-company-link" id="linkHelp" href="terms-conditions" target="_blank" onclick="return false;">terms and conditions
                     <span class="visually-hidden">will open in a new tab</span>
                   </a>
                 </label>
               </div>
             </div>
     
-            <div class="field">
+            <div class="field" role="radiogroup">
```

Comment:
I think it will be better add fieldset with legend. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role